docs/docs/cryptography.md

```diff
@@ -138,19 +138,42 @@ Additionally, Lagrange interpolation also guarantees that no set of less than $t
 
 In summary, a threshold BLS signature $\sigma$ exhibits all properties required for publicly-verifiable, unbiased, unpredictable, and distributed randomness.
 
+#### Smaller signatures for resource constrained applications
+
+The above description of the signature threshold scheme accurately describes how we generate signatures that contain all the necessary properties for randomness under the `pedersen-bls-chained` and `pedersen-bls-unchained` schemes. 
+Under that scheme, messages are mapped to a point on $\mathbb{G}_1$ and public keys are a point on $\mathbb{G}_2$. As a result, message signatures are 96 bytes long and public keys are 48 bytes long. For systems where users are signing large messages and/or storing a lot of public keys, this trade-off makes a lot of sense: a 96 byte signature will be a small proportion of the entire message stored or transmitted. Additionally, signatures can be aggregated to save space.
```

Comment:
```suggestion
Under these schemes, messages are mapped to a point on $\mathbb{G}_2$ and public keys are a point on $\mathbb{G}_1$. As a result, message signatures are 96 bytes long and public keys are 48 bytes long. For systems where users are signing large messages and/or storing a lot of public keys, this trade-off makes a lot of sense: a 96 byte signature will be a small proportion of the entire message stored or transmitted. Additionally, signatures can be aggregated to save space, whereas aggregation isn't usually desirable for public keys, thus minimizing the size of the public keys is optimal for most systems.
```

Reply:
Disgusting use of american english, but I'll let it slide this time >.>